addons/ai/functions/fnc_rf_returnFire.sqf

```diff
@@ -0,0 +1,33 @@
+ 
+/*
+    Author:
+        Adam Bridgford, Tim Beswick
+
+    Description:
+        Makes unit fire 20 rounds fast.
+        
+    Parameter(s):
+        0: Unit hit <OBJECT>
+        1: Unit causing the damage <SCALAR>
+        
+    Return Value:
+        Nothing
+*/
+ 
+ params ["_unit","_instigator"];
+ 
+        _wpn = currentWeapon _unit;
+        hint format ["%1",_wpn];
+        _unit doTarget _instigator;
+        _unit Fire _wpn;
+        shots = 0;
+        hint format ["Shots: %1",shots];
+        _unit addEventHandler ["fired", {
```

Comment:
Has this been tested? I'm not sure of the logic here. Is this just acting as a counter to force full auto? Or is it aiming to actually make the guy fire 20 times on full auto?